components/ListingsStarshipCard/ListingsStarshipCard.js

```diff
@@ -0,0 +1,79 @@
+import useSWR from "swr";
+import Image from "next/image";
+import styled from "styled-components";
+import Flex from "../Layout/Flex";
+
+const StyledImageWrapper = styled.div`
+  height: 84px;
+  width: 84px;
+  border-radius: 50%;
+  overflow: hidden;
+  position: relative;
+`;
+
+const StyledCard = styled.div`
+  border-radius: 11px;
+  border: 0.5px solid var(--secondary-color);
+  padding: 1rem;
+  background-color: var(--button-color));
+`;
+
+const StyledStarshipImage = styled(Image)`
+  object-fit: cover;
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  margin: auto;
+`;
+
+const StyledParagraph = styled.p`
+  color: var(--secondary-color);
+`;
+const StyledParagraphPreis = styled.p`
+  color: var(--secondary-color);
+  font-size: 2rem;
+`;
+
+const StyledBuyButton = styled.button`
+  background-color: #000000;
+  border-radius: 5px;
+  border: 0.3px solid var(--secondary-color);
+  height: 2rem;
```

Comment:
At the top you are working with px for height and width! For better readability and a uniform look of the code it would be better to stick to one unit :)